internal/planner/planner.go

```diff
@@ -432,7 +432,7 @@ func (p *Planner) planQueries() error {
 
 		for _, qv := range qvs.Sorted() {
 			qv = rewrittenVar(qs.RewrittenVars, qv)
-			if !qv.IsGenerated() && !qv.IsWildcard() {
```

Comment:
Clarification question: this is removed because there are not wildcards finding their way into queries anymore with this change, is that correct?

Reply:
Updated.